azure-pipelines.yml

```diff
@@ -0,0 +1,74 @@
+pool:
+  name: VSEng-MicroBuildVS2017
+  demands: msbuild
+
+variables:
+  BuildConfiguration: 'Release'
+  BuildPlatform: ''
+
+steps:
+- task: MicroBuildSigningPlugin@2
+  inputs:
+    signType: '$(SignType)'
+    zipSources: true
+    feedSource: 'https://devdiv.pkgs.visualstudio.com/DefaultCollection/_packaging/MicroBuildToolset/nuget/v3/index.json'
+
+- task: MSBuild@1
+  displayName: '/t:Pack'
+  inputs:
+    solution: MSBuildLocator.sln
+    platform: '$(BuildPlatform)'
+    configuration: '$(BuildConfiguration)'
+    msbuildArguments: '/t:Pack /restore -binaryLogger:$(Build.SourcesDirectory)\Pack.binlog'
+
+- task: CopyFiles@2
+  displayName: 'Collect Symbols'
+  inputs:
+    SourceFolder: src/MSBuildLocator
+    Contents: '**\*.pdb'
+    TargetFolder: '$(Build.ArtifactStagingDirectory)/symbols'
```

Comment:
slashes seem to be going both ways. This is windows, right?

Reply:
Yeah, our official build is on Windows, but I don't think this matters.